content/guides/hosting/hosting-options/self-managed/install-on-public-cloud/aws-tf.md

```diff
@@ -59,6 +59,7 @@ The steps on this topic are common for any deployment option covered by this doc
    zone_id                    = "xxxxxxxxxxxxxxxx"
    allowed_inbound_cidr       = ["0.0.0.0/0"]
    allowed_inbound_ipv6_cidr  = ["::/0"]
+   eks_cluster_version        = "1.29"
```

Comment:
Do you have a plan for keeping this up to date?

Reply:
Most of this doc I created in the past because we had nothing, but I don't keep them up-to-date.
What I do is mostly fix things when I notice they are not correct.
For this particular one, I can commit to finish the updates.
In the long run, it would be nice if the team that releases the updates keeps this doc also up-to-date.